nih-sparc/sparc-app
# Organ detail page - Display related projects #4a786r

# Description

The purpose of this PR is to display the related projects for an organ. The organ detail page was not currently returning the data from contentful to the Vue component, so I also did that.

## Type of change


# How Has This Been Tested?

- Visit the Find Data page
- Click on the 'Organs' tab
- Click on the 'Heart' organ
- You should be taken to that organ's page
- Click on the 'Projects' tab
- You should see the related projects to that organ
- Click on one of the projects names
- You should be taken to that project's page

# Checklist:
